_candidates_ja/LucDuponcheel_1.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Luc Duponcheel
+title: "記述的な方法による関数型プログラミング"
+length: 40
+audience: Advanced
+language: English
+twitter: LucDup
+github: LucDupAtGitHub
+icon: https://avatars0.githubusercontent.com/u/77954?s=400&u=7f67fc0da57a4ef97a65349c4f7cd39f40557812&v=4
+organization: ImagineJ
+tags:
+  - Functional Programming
+  - Best Practices
+suggestions:
+  - このトークを聞けば参加者は、高レベルな記述部分(description)と低レベル(でアクティブもしくはリアクティブ)な実行部分(interpretation)とを分離することによって実現されるプログラムの柔軟性を理解することが出来ます
```

Comment:
After attending the talk the attendee will appreciate the flexibility of separating program descriptions from their executable interpretations (active, reactive, ...).